src/commands/init.js

```diff
@@ -159,8 +159,12 @@ module.exports = function (id, source) {
         // write out files
         .then(function (answers) {
             if (answers.confirmCreation) {
-                if (fs.exists('.pullquester')) {
```

Comment:
we should probably switch these to async requests since fs.xxxSync instances are deprecated.